geps/gep-1619/index.md

```diff
@@ -317,71 +346,55 @@ In this section, we will explore the questions and design elements associated wi
 ### GO
 
 ```go
-// SessionPersistencePolicy provides a way to define session persistence rules
-// for a service or route.
+// LoadBalancerPolicy provides a way to define load balancing rules
```

Comment:
Naming 🧵 for anyone interested.

I agree with the idea to use a more generic/broad term here, but I'm not 100% sure what we want that name to be. To help frame this, here are some ideas that may fit into a policy attached to a Service:

1. Session Affinity
2. Timeout
3. Connection Draining
4. Logging
5. LB Algorithm

Of course this does get more complex if we want to also attach this to Route in the future, so we _probably_ don't want "Backend" in the name of this policy.

Some ideas beyond LoadBalancerPolicy:

* LBPolicy
* LoadBalancingPolicy
* TrafficPolicy
* Other ideas welcome

For the record, I don't hate LoadBalancerPolicy, just thought I'd throw out some other names in case any stuck.

Reply:
I don't mind LBPolicy and LoadBalancerPolicy. Agree it's better to not have 'backend' in the name - but I don't mind it too much, setting load balancing policies per route is not common and  unlikely to support all options and semantics. So if/when that is desired - a RouteBalancerPolicy may be added with a small subset.

Reply:
This is an interesting pattern I hadn't considered yet. Policy for Service, and Inline for Route. I had some confusion on exactly what route attachment meant as far as inherited policy attachment design, but this approach seems simpler to me, i.e. it seems problematic to start with Direct Policy service attachment, then later having to redesign into Inherited Policy attachment to also support Routes.

The configuration, though common, would be in two different places (Policy & Route), so that may be considered a negative, but the positive is that is it gives us more independence to design LoadBalancerPolicy for services.

I'm in favor, but would like to hear other opinions.